### PR TITLE
5.2.3 version history

### PR DESCRIPTION
**Pull request by hflynn**
Will be staged at https://www.openmicroscopy.org/site/support/omero5.2-staging/users/history.html once the autogen branch has been built.

### CODE REVIEW

File: history.txt [hunk 1]
```diff
@@ -1,6 +1,40 @@
...
+A bug-fix release. Improvements include:
+
+-  fixed problem with float images
+-  all scripts currently exposed to users via our website have been reviewed
+   and fixed where necessary so they are all now 5.2.x compatible, and a new
+   omero-install workflow has been developed to ensure these are reviewed
+   regularly going forward
+-  better support for metadata annotations in clients including tag/tagset
+   support and performance issues
+-  fixes in OMERO.web for deleting mifs
```
Comment (mtbc) [inline history.txt]: might be MIFs?

Comment (hflynn) [inline history.txt]: Good point, I'll fix that once I've collected content comments

**Comment by jburel** [comment]
* Need to specify ice version 3.6.2 only

**Comment by hflynn** [comment]
I hadn't noticed Travis had failed, that wouldn't have helped my efforts to get this built on the docs!

**Comment by jburel** [comment]
```required dependencies now fully tested```. Maybe better to write ```review required dependencies``` (or we don't mention it)

**Comment by hflynn** [comment]
I'd default to not mention it given the ongoing discussions!

**Comment by jburel** [comment]
```Ice 3.6.2 support```: not on windows.

**Comment by jburel** [comment]
fixes in OMERO.web for deleting MIFs
maybe add a line "other bug fixes in web"

**Comment by hflynn** [comment]
Should be sorted now @jburel

**Comment by aleksandra-tarkowska** [comment]
sorry for got to add that, we clarified apache startup

**Comment by aleksandra-tarkowska** [comment]
also:
 - allows nginx servername pass directly from cli
 - allows storing web static files in custom location

for developers:
 - added ability to run robot test using remote driver

**Comment by jburel** [comment]
maybe something "improve web configuration" rather than a detailed of the specific PR (e.g. robot one is mainly of internal usage)

**Comment by jburel** [comment]
Thanks @hflynn.
Merging

**Pull request merged**